### PR TITLE
New Branch

### PR DESCRIPTION
**Pull request by abhirupr123**
## DIKSHA

Digital Learners Passbook

### Changes

Week 1 and 2 updates along with author name changes 

### Notes

This is a new branch after making changes in previous branches.

### CODE REVIEW

File: docs/2023/DIKSHA/Digital Learners Passbook/updates/2023-09-01.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+- [ ] Give the description about Milestone 1
+- [ ] Give the description about Milestone 2
+- [ ] Give the description about Milestone 3
+- [ ] Give the description about Milestone 4
+
+## Screenshots / Videos 
+
+## Contributions
+
+## Learnings
```
Comment (Shruti3004) [inline docs/2023/DIKSHA/Digital Learners Passbook/updates/2023-09-01.md:16]: @abhirupr123 please fix the name of images to a proper one

**Review by Shruti3004 — CHANGES_REQUESTED** [review]

File: docs/2023/DIKSHA/Digital Learners Passbook/updates/2023-09-01.md (hunk 1 above)
Comment (abhirupr123) [reply docs/2023/DIKSHA/Digital Learners Passbook/updates/2023-09-01.md:16]: > @abhirupr123 please fix the name of images to a proper one

I have changed the image and video names

**Review by photon0205 — CHANGES_REQUESTED** [review]
As you are having the static folder, it would be better if you could move all the images and videos to that folder

File: docs/2023/DIKSHA/Digital Learners Passbook/updates/2023-07-21.md
[the remark this answers is not in the document]
Comment (abhirupr123) [reply docs/2023/DIKSHA/Digital Learners Passbook/updates/2023-07-21.md:1]: > add week 3 updates

added week 3 updates and project description

**Review by photon0205 — CHANGES_REQUESTED** [review]
Keep a sensible/ logical alt names for the screenshots

**Comment by abhirupr123** [comment]
> Keep a sensible/ logical alt names for the screenshots

Done

**Pull request merged**